### ISSUE TITLE
find missing number in array

### ISSUE THREAD

**Issue by songz** [llipio/algorithms#18]
return missing number in an input array of consecutively increasing numbers.

i: [5,6,8,9]
o: 7

**Comment by Himanshu1495** [comment]
Can i try this one ?

**Comment by songz** [comment]
yes @Himanshu1495 , but you would have to do it in your own environment. :)

**Comment by Himanshu1495** [comment]
Can you please review the solution @songz ?
